plugins/modules/mysql_user.py

```diff
@@ -156,6 +156,15 @@
     type: dict
     version_added: '3.6.0'
 
+  column_case_sensitive:
+    description:
+      - When C(true), the module won't uppercase the field names in the privileges.
+      - When C(false), the field names will be upper cased. This is the default
+      - This feature was introduced because MySQL 8 and above uses case sensitive
+        fields names in privileges.
```

Comment:
```suggestion
    description:
      - The default is C(false).
      - When C(true), the module will not uppercase the field names in the privileges.
      - When C(false), the field names will be upper-cased. This is the default
      - This feature was introduced because MySQL 8 and above uses case sensitive
        fields names in privileges.
```